cpp/ycm/ClangCompleter/ClangUtils.h

```diff
@@ -21,6 +21,13 @@
 #include <clang-c/Index.h>
 #include <string>
 
+#if defined(CINDEX_VERSION_MAJOR) && defined(CINDEX_VERSION_MINOR) && \
+    (CINDEX_VERSION_MAJOR > 0 || CINDEX_VERSION_MINOR >= 6)
```

Comment:
Yes, in libclang 3.5 the `CINDEX_VERSION_MINOR` is 27. I will remove the checks.